langwrappers/src/main/kotlin/org/jetbrains/research/testspark/langwrappers/PsiComponents.kt

```diff
@@ -148,7 +150,7 @@ interface PsiHelper {
      *         The array contains the class display name, method display name (if present), and the line number (if present).
      *         The line number is prefixed with "Line".
      */
-    fun getCurrentListOfCodeTypes(e: AnActionEvent): Array<*>?
+    fun getCurrentListOfCodeTypes(e: AnActionEvent): List<Pair<CodeType, String>>?
```

Comment:
As wrote previously, this `Pair<CodeType, String>` should be aliased for better clarity.

Does it have to be `List<..>?`? If things go wrong, we can return an empty list; would it be a better approach, or is it intentional here?

Reply:
Right, no need